dataactbroker/handlers/accountHandler.py

```diff
@@ -447,7 +450,7 @@ def changeStatus(self,system_email):
     def listUsers(self):
         """ List all users ordered by status. Associated request body must have key 'filter_by' """
         user = self.interfaces.userDb.getUserByUID(LoginSession.getName(flaskSession))
-        isAgencyAdmin = True if self.interfaces.userDb.hasPermission(user, "agency_admin") else False
+        isAgencyAdmin = self.userManager.hasPermission(user, "agency_admin") and not self.userManager.hasPermission(user, "website_admin")
```

Comment:
Just to clarify: do we know for sure that an agency admin will never also be a website admin?